sdcm/cluster.py

```diff
@@ -4491,7 +4491,7 @@ def get_test_keyspaces(self, db_node=None):
         """Function returning a list of non-system keyspaces (created by test)"""
         db_node = db_node or self.nodes[0]
         keyspaces = db_node.run_cqlsh("describe keyspaces").stdout.split()
-        return [ks for ks in keyspaces if not is_system_keyspace(ks)]
+        return [ks.strip('"') for ks in keyspaces if not is_system_keyspace(ks)]
```

Comment:
we do it also in `nemesis.py` in line 2922, could be removed from there

Reply:
probably will cause failure for case in line `nemesis.py:4153`: need to add `cql_quote_if_needed`

Reply:
I'm not sure if we shouldn't add `cql_quote_if_needed` for `sdcm.cluster.BaseNode.get_cfstats` - requires verification which form fits it